### PR TITLE
9987 kill quadratic lands remove app

### PR DESCRIPTION
**Pull request by schultztimothy**
##### Description

This is a follow up to https://github.com/gitcoinco/web/pull/10137. It removes the remaining files and reference to quadratic lands

##### Refers/Fixes

fixes: #9987

### CODE REVIEW

File: app/app/urls.py [hunk 1]
```diff
@@ -770,9 +771,9 @@
     url(r'^blocknative', perftools.views.blocknative, name='blocknative'),
 
     # quadratic lands
-    path('quadraticlands/', include('quadraticlands.urls', namespace='quadraticlands')),
-    re_path(r'^quadraticlands/?', include('quadraticlands.urls', namespace='ql_catchall_')),
-    re_path(r'^quadraticland/?', include('quadraticlands.urls', namespace='ql_catchall')),
```
Comment (thelostone-mc) [inline app/app/urls.py:775]: this is just a nightmare :P meh meh meh

File: Dockerfile [hunk 2]
```diff
@@ -24,7 +24,6 @@ WORKDIR /usr/src
 RUN apt-get update && apt-get install -y wget
 RUN wget "https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
 RUN apt-get install -y $CHROME_DEPS
-RUN dpkg -i google-chrome-stable_current_amd64.deb
```
Comment (gdixon) [inline Dockerfile:27]: Was this supposed to be dropped here?

Comment (thelostone-mc) [reply Dockerfile:27]: lol that fails on the m1 :P but should be a in a seperate PR anyways

**Comment by thelostone-mc** [comment]
@schultztimothy the other PR is rolled out ! When you're ready. -> we can update this PR and make sure only the new commits are present

**Pull request closed without merging**